src/ConnectionConfig.tsx

```diff
@@ -36,13 +36,13 @@ export const ConnectionConfig: FC<ConnectionConfigProps> = (props: ConnectionCon
   if (profile === undefined) {
     profile = options.database;
   }
-
-  const settings = (window as any).grafanaBootData.settings;
+  
+  const awsAssumeRoleEnabled = config.awsAssumeRoleEnabled
+  const awsAllowerProvidersConfig = config.awsAllowedAuthProviders as AwsAuthType[]
```

Comment:
I think we can use a type guard instead of a type assertion.

Maybe something like this at the top of the file.

```ts
const isAwsAuthType = (value: any): value is AwsAuthType => {
  return typeof value === 'string' && awsAuthProviderOptions.some((opt) => opt.value === value);
}
```

And then for this line something like this.

```ts
const awsAllowerProvidersConfig = config.awsAllowedAuthProviders.filter(isAwsAuthType);
```

Reply:
Thanks for the suggestion, done!